service-commands/scripts/provision-dev-env.sh

```diff
@@ -68,7 +68,7 @@ function setup_python() {
     sudo add-apt-repository ppa:deadsnakes/ppa # python3.9 installation
     sudo apt install -y "python$PYTHON_VERSION"
     sudo apt install -y "python$PYTHON_VERSION-dev"
-    pip install wheel
+    pip install wheel yq
```

Comment:
Moved this install to this PR: https://github.com/AudiusProject/audius-protocol/pull/2592/files

The change to requirements.txt can't be moved since it's the lib that the rest of this PR relies on.